tee-worker/ts-tests/identity-direct-invocation.test.ts

```diff
@@ -0,0 +1,62 @@
+import { KeyObject } from 'crypto';
+import { step } from 'mocha-steps';
+import { assert } from 'chai';
+import { hexToU8a } from '@polkadot/util';
+
+import { initIntegrationTestContext } from './common/utils';
+import {
+    createSignedTrustedGetterUserShieldingKey,
+    getTEEShieldingKey,
+    sendRequestFromTrustedGetter,
+} from './examples/direct-invocation/util'; // @fixme move to a better place
+import { IntegrationTestContext } from './common/type-definitions';
+
+describe('Test Identity (direct invocation)', function () {
+    let context: IntegrationTestContext = undefined as any;
+    let teeShieldingKey: KeyObject = undefined as any;
```

Comment:
Doing these here for now since I didn't want to mess with `describeLitentry` just yet; will figure a better way later. Maybe we can just use Jest instead of Mocha and then we'd have async support in `describe`?